### PR TITLE
LPS-168816 Add the frontend component for the Scope Card in the Segments Editor

### PR DESCRIPTION
**Pull request by Stephy6**
Motivation
=======
We are moving the Segmentation from a site to a global scope. In order to that we need to introduce a tool to manage sites and assign them to Segments. We are using a Scope Card to select sites for this purpose. This PR contains the Front-end part with no site selector, just the Scope Card with the current site.
[Link to the taskt](https://issues.liferay.com/browse/LPS-168816)
[Link to the story](https://issues.liferay.com/browse/LPS-167025)

Proposed Solution
========
The proposed solution consists in using a ClayPanel to render the Card, where the PanelHeader contains the title "Scope" and the PanelBody contains the Site Name.

Steps to Verify:
----------------
1. Go to People -> Segments.
2. Click on '+' to add a new Segment.
3. Check that the Scope Card is displayed correctly.

Screencaps
---------------
Captura de pantalla 2022-11-23 a las 11 32 18

### CODE REVIEW

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_contributor_builder.scss [hunk 1]
```diff
@@ -35,7 +35,13 @@
 				transition: all 0.5s ease;
 
 				.content-wrapper {
-					padding: 40px 24px;
+					padding: 18px 24px;
```
Comment (claraizquierdo) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_contributor_builder.scss:38]: In the figma I see a padding top of 24px. Maybe you can try to use a class for applying the padding and remove this css line.

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js [hunk 2]
```diff
@@ -161,6 +162,42 @@ class ContributorBuilder extends React.Component {
 
 							<ClayLayout.ContainerFluid>
 								<div className="content-wrapper">
+									{Liferay.FeatureFlags['LPS-166954'] && (
+										<ClayPanel
+											collapsable
+											defaultExpanded={true}
+											displayTitle={
+												<h2 className="mb-0 size- text-dark">
```
Comment (claraizquierdo) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js:170]: The class `size-` is not correct.  Also, maybe it is good to use the ClayPanel.Header component.

**Review by claraizquierdo — CHANGES_REQUESTED** [review]
Ei @Stephy6, good work so far. I left a few comments in the code. Also, I think that the margin bottom of the scope card is not correct.

**Review by claraizquierdo — APPROVED** [review]
Ei @Stephy6 I have reordered the commits and applied a few changes to the styles in order to follow the designs. Could have a look a the changes? If you see that they are correct feel free to forward this PR.

**Comment by claraizquierdo** [comment]
ci:forward

**Comment by claraizquierdo** [comment]
ci:test:sf

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by claraizquierdo** [comment]
ci:test:tango-acceptance

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by Stephy6** [comment]
Hey @cgoncas @claraizquierdo I've just dropped the buildLang commit. Checked and LGTM 🎉 !

**Comment by cgoncas** [comment]
ci:test:sf

**Review by cgoncas — APPROVED** [review]
LGTM

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:tango

**Review by claraizquierdo — APPROVED** [review]
I added a commit removing an unnecessary translation. Everything else is working fine 👍

**Comment by claraizquierdo** [comment]
@CarlosBrichete could you have a look at the failing tests please?

**Comment by cgoncas** [comment]
ci:forward

**Comment by cgoncas** [comment]
ci:reevaluate:1357401_3119

**Comment by CarlosBrichete** [comment]
Hey, as @cgoncas said, the failures are not related to this PR :)

**Comment by claraizquierdo** [comment]
ci:reevaluate:1357401_3119

**Comment by Stephy6** [comment]
ci:forward

**Comment by Stephy6** [comment]
ci:stop

**Comment by claraizquierdo** [comment]
ci:test:sf

**Comment by claraizquierdo** [comment]
ci:forward

**Comment by Stephy6** [comment]
ci:forward

**Comment by claraizquierdo** [comment]
ci:reevaluate:1337401_2501

**Comment by Stephy6** [comment]
ci:forward

**Comment by Stephy6** [comment]
ci:stop

**Comment by claraizquierdo** [comment]
ci:stop

**Comment by claraizquierdo** [comment]
Ei @Stephy6 , I think that the LSP of the branch name is not correct. It should be LPS-168816. There is also one commit with the wrong LPS (my bad). Do you mind changing it?

**Pull request closed without merging**